packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -72,8 +76,18 @@ const cicdContent = (
   // This is an example of how you can implement your company's logic in entity page.
   // You can for example enforce that all components of type 'service' should use GitHubActions
   <EntitySwitch>
-    <EntitySwitch.Case if={isGithubActionsAvailable}>
-      <EntityGithubActionsContent />
+    <EntitySwitch.Case if={isEitherTerraformOrGitubActionsAvailable}>
```

Comment:
I can understand why you've done it this way, but one of the main issues with this is that our package shouldn't know which other plugins are being used. If we were to add another plugin here, we'd need to update the plugin with a new helper method which isn't quite the right process.

The EntityPage is within the control of the consumer and they can implement whatever logic they wish to, we may be doing it one way, but another consumer could choose to do it another which is why I think it's important to just expose only the `isTerraformAvailable` helper method from our plugin.

Going forward, I think this should be as follows (similar to how Spotify do the `entityWarningContent`):
```jsx
    <>
      <EntitySwitch>
        <EntitySwitch.Case if={isGithubActionsAvailable}>
          <EntityGithubActionsContent />
        </EntitySwitch.Case>
      </EntitySwitch>

      <EntitySwitch>
        <EntitySwitch.Case if={isTerraformAvailable}>
          <EntityTerraformLatestRunCard />
        </EntitySwitch.Case>
      </EntitySwitch>

      <EntitySwitch>
        <EntitySwitch.Case if={!isGithubActionsAvailable && !isTerraformAvailable}>
          <EmptyState />
        </EntitySwitch.Case>
      </EntitySwitch>
    <>
```